orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/entity/VectorOfTrust.java

```diff
@@ -69,10 +69,30 @@ public static VectorOfTrust parseFromAuthRequestAttribute(List<String> vtr) {
             LOG.warn("Error when parsing vtr attribute", e);
             throw new IllegalArgumentException("Invalid VTR attribute", e);
         }
-        VectorOfTrust vectorOfTrust = parseVtrSet(vtrJsonArray);
-        LOG.info("VTR has been processed at vectorOfTrust: {}", vectorOfTrust);
+        List<VectorOfTrust> vtrList = parseVtrSet(vtrJsonArray);
+        LOG.info("VTR has been processed at vectorOfTrust: [{}]", stringifyVtrList(vtrList));
 
-        return vectorOfTrust;
+        return vtrList.get(0);
```

Comment:
We've lost all the sorting and ordering that was in the `parse` method and now just return the first at random?